web/js/services.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+angular.module('talklyApp.services', ['ngResource'])
+    .factory('Topics', ['$resource', function ($resource) {
+        return $resource('/api/topics', {}, {
```

Comment:
This only works, if you have a special vhost setup for talkly. It does not work, if the app is served in a subfolder.